### ISSUE TITLE
hi,Kernel crashes sometimes occur. What causes them?

### ISSUE THREAD

**Issue by micenote** [DmitriyyyyS/Asus-Prime-Z370-P#4]
error information:
panic(cpu 2 caller 0xffffff800604c32a): Kernel trap at 0xffffff8005efd8d1, type 14=page fault, registers:
CR0: 0x0000000080010033, CR2: 0x0000000000000435, CR3: 0x00000003bcb9d02b, CR4: 0x00000000003626e0
RAX: 0x0000000000000134, RBX: 0x0000000000000074, RCX: 0x00000000000000dc, RDX: 0xffffff80068094b0
RSP: 0xffffff92154732b2, RBP: 0xffffff9215473e08, RSI: 0x0000000000000001, RDI: 0xffffff9215473e10
R8:  0x0000000000000000, R9:  0x0000000000000000, R10: 0x000000000000001c, R11: 0xffffff9215473e20
R12: 0x000000001000000d, R13: 0x00000000000000dc, R14: 0xffffff9215473ec0, R15: 0x00007000049f4ff8
RFL: 0x0000000000010202, RIP: 0xffffff8005efd8d1, CS:  0x0000000000000008, SS:  0x0000000000000010
Fault CR2: 0x0000000000000435, Error code: 0x0000000000000002, Fault CPU: 0x2, PL: 0, VF: 0

Backtrace (CPU 2), Frame : Return Address
0xffffff9215472d10 : 0xffffff8005f215cd 
0xffffff9215472d60 : 0xffffff800605a3c5 
0xffffff9215472da0 : 0xffffff800604bf7e 
0xffffff9215472df0 : 0xffffff8005ec7a40 
0xffffff9215472e10 : 0xffffff8005f20c97 
0xffffff9215472f10 : 0xffffff8005f21087 
0xffffff9215472f60 : 0xffffff80066c2c7c 
0xffffff9215472fd0 : 0xffffff800604c32a 
0xffffff9215473150 : 0xffffff800604c028 
0xffffff92154731a0 : 0xffffff8005ec7a40 
0xffffff92154731c0 : 0xffffff8005efd8d1 
0xffffff9215473e08 : 0x134 
0xffffff9215473e70 : 0xffffff8005f149b4 
0xffffff9215473f00 : 0xffffff8006032605 
0xffffff9215473fa0 : 0xffffff8005ec8226 

BSD process name corresponding to current thread: World of Warcraf
Boot args: shikigva=80

Mac OS version:
19E287

Kernel version:
Darwin Kernel Version 19.4.0: Wed Mar  4 22:28:40 PST 2020; root:xnu-6153.101.6~15/RELEASE_X86_64
Kernel UUID: AB0AA7EE-3D03-3C21-91AD-5719D79D7AF6
Kernel slide:     0x0000000005c00000
Kernel text base: 0xffffff8005e00000
__HIB  text base: 0xffffff8005d00000
System model name: iMac19,2 (Mac-63001698E7A34814)
System shutdown begun: NO
Panic diags file available: YES (0x0)

System uptime in nanoseconds: 64798975716601
last loaded kext at 57194108333743: @macos.driver.!UEthernetHost	8.1.1 (addr 0xffffff7f8ae87000, size 49152)
last unloaded kext at 57255415755458: >!UMergeNub	900.4.2 (addr 0xffffff7f8ae84000, size 12288)
loaded kexts:
com.parallels.kext.netbridge	14.1.1 45476
com.parallels.kext.hypervisor	14.1.1 45476
com.parallels.kext.vnic	14.1.1 45476
com.insanelymac.RealtekRTL8111	2.3.0d7
ru.joedm.SMCSuperIO	1.1.2
as.vit9696.SMCProcessor	1.1.2
as.vit9696.VirtualSMC	1.1.2
org.acidanthera.NVMeFix	1.0.2
com.rehabman.driver.USBInjectAll	0.7.3
as.vit9696.!AALC	1.4.8
as.vit9696.WhateverGreen	1.3.9
as.vit9696.Lilu	1.4.4
@macos.driver.!UEthernetHost	8.1.1
@filesystems.smbfs	3.4.2
>AudioAUUC	1.70
>!AUpstreamUserClient	3.6.8
>!AMCCSControl	1.11
@kext.AMDFramebuffer	3.0.8
@fileutil	20.036.15
>AGPM	111.4.4
>!APlatformEnabler	2.7.0d0
>X86PlatformShim	1.0.0
@filesystems.autofs	3.0
@filesystems.ntfs	3.14.3
>!AHDA	283.15
@kext.AMDRadeonX4000	3.0.8
@kext.AMDRadeonServiceManager	3.0.8
@AGDCPluginDisplayMetrics	5.1.16
>!AHV	1
|IOUserEthernet	1.0.1
|IO!BSerialManager	7.0.4f6
@private.KextAudit	1.0
>pmtelemetry	1
>!A!IKBLGraphics	14.0.5
@Dont_Steal_Mac_OS_X	7.0.0
>!A!ISlowAdaptiveClocking	4.0.0
@kext.AMD9500!C	3.0.8
>!A16X50ACPI	3.2
>!A!IPCHPMC	2.0.1
>!AGFXHDA	100.1.426
>!A!ICFLGraphicsFramebuffer	14.0.5
>!AVirtIO	1.0
@filesystems.hfs.kext	522.100.5
@!AFSCompression.!AFSCompressionTypeDataless	1.0.0d1
@BootCache	40
@!AFSCompression.!AFSCompressionTypeZlib	1.0.0
@filesystems.apfs	1412.101.1
>!AAHCIPort	341.0.2
>!ARTC	2.0
>!AACPIButtons	6.1
>!AHPET	1.8
>!ASMBIOS	2.1
>!AACPIEC	6.1
>!AAPIC	1.7
$!AImage4	1
@nke.applicationfirewall	303
$TMSafetyNet	8
@!ASystemPolicy	2.0.0
|EndpointSecurity	1
>!ASMBus!C	1.0.18d1
@kext.triggers	1.0
>DspFuncLib	283.15
@kext.OSvKernDSPLib	529
@kext.AMDRadeonX4000HWLibs	1.0
@kext.AMDRadeonX4000HWServices	3.0.8
|IOAVB!F	840.3
>!ASSE	1.0
|IO!B!F	7.0.4f6
|IO!BPacketLogger	7.0.4f6
@!AGPUWrangler	5.1.16
|IOSlowAdaptiveClocking!F	1.0.0
>!A16X50Serial	3.2
@kext.AMDSupport	3.0.8
|IONDRVSupport	575.1
>X86PlatformPlugin	1.0.0
>IOPlatformPlugin!F	6.0.0d8
@!AGraphicsDeviceControl	5.1.16
|IOAccelerator!F2	438.4.5
>!AHDA!C	283.15
|IOGraphics!F	575.1
|IOHDA!F	283.15
@plugin.IOgPTPPlugin	840.3
|IOEthernetAVB!C	1.1.0
|IOSkywalk!F	1
>usb.networking	5.0.0
>usb.!UHostCompositeDevice	1.2
|IOAudio!F	300.2
@vecLib.kext	1.2.0
|IOSerial!F	11
|IOSurface	269.11
@filesystems.hfs.encodings.kext	1
|IOAHCIBlock!S	316.100.5
|IOUSB!F	900.4.2
>!AEFINVRAM	2.1
|IOAHCI!F	290.0.1
>usb.!UXHCIPCI	1.2
>usb.!UXHCI	1.2
>!AEFIRuntime	2.1
|IOSMBus!F	1.1
|IOHID!F	2.0.0
$quarantine	4
$sandbox	300.0
@kext.!AMatch	1.0.0d1
>DiskImages	493.0.0
>!AFDEKeyStore	28.30
>!AEffaceable!S	1.0
>!AKeyStore	2
>!UTDM	489.101.1
|IOSCSIBlockCommandsDevice	422.101.1
>!ACredentialManager	1.0
>KernelRelayHost	1
>!ASEPManager	1.0.1
>IOSlaveProcessor	1
|IOUSBMass!SDriver	157.101.3
|IOSCSIArchitectureModel!F	422.101.1
|IO!S!F	2.1
|IOUSBHost!F	1.2
>!UHostMergeProperties	1.2
>usb.!UCommon	1.0
>!ABusPower!C	1.0
|CoreAnalytics!F	1
>!AMobileFileIntegrity	1.0.5
@kext.CoreTrust	1
|IOTimeSync!F	840.3
|IONetworking!F	3.4
|IOReport!F	47
>!AACPIPlatform	6.1
>!ASMC	3.1.9
>watchdog	1
|IOPCI!F	2.9
|IOACPI!F	1.4
@kec.pthread	1
@kec.corecrypto	1.0
@kec.Libm	1

**Comment by DmitriyyyyS** [comment]
World of Warcraf :::::: It seems that some kind of running process breaks the system and causes panic.

**Comment by DmitriyyyyS** [comment]
To see why and what panic you need to register the keys -v debug = 0x100 keepsyms = 1 and enable the patch (Quirks) PanicNoKextDump

**Comment by micenote** [comment]
> World of Warcraf :::::: It seems that some kind of running process breaks the system and causes panic.

yes,run world of warcraf for a period of time.macOS is crashed and reboot
I suspect the graphics card is overheated.
normal use it ,no problem.
